### PR TITLE
restructure languages

### PR DESCRIPTION
**Pull request by sesi200**
Thank you for your contribution to the IC Developer Portal.
Before submitting your Pull Request, please make sure that:

- [x] You have updated the [`.github/CODEOWNERS`](https://github.com/dfinity/portal/blob/master/.github/CODEOWNERS) file according to your changes.
- [x] You have registered new documents to [`/sidebars.js`](https://github.com/dfinity/portal/blob/master/sidebars.js) if any.

### CODE REVIEW

File: docs/samples/nft.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ The NFT canister is not very complicated since the [DIP-721](https://github.com/
 but we can still use it to explain three important concepts concerning dapp development for the Internet Computer:
 
 ### Stable Memory for Canister Upgrades
-The Internet Computer employs [Orthogonal Persistence](https://smartcontracts.org/docs/current/developer-docs/build/languages/motoko#orthogonal-persistence), so developers generally do not need to think a lot about storing their data.
+The Internet Computer employs [Orthogonal Persistence](https://smartcontracts.org/docs/current/developer-docs/build/cdks/motoko-dfinity#orthogonal-persistence), so developers generally do not need to think a lot about storing their data.
```
Comment (olaszakos) [inline docs/samples/nft.md:22]: URL is broken and old, please make it point to an .md file

Comment (sesi200) [reply docs/samples/nft.md]: fixed

**Review by olaszakos — APPROVED** [review]

**Review by dprats — APPROVED** [review]

File: docs/developer-docs/build/agents/index.md [hunk 2]
```diff
@@ -1,17 +1,30 @@
...
+
+This section of the docs covers the following agents, ordered by languages:
+- JavaScript
+  - [`agent` by DFINITY](./agent-dfinity.md)
+- Rust
+  - [`ic-agent` by DFINITY](./ic-agent-dfinity.md)
+- TypeScript
+  - [`agent` by DFINITY](./agent-dfinity.md)
+
+Besides those, there exist a lot of other community-supported agents:
```
Comment (ericswanson-dfinity) [inline docs/developer-docs/build/agents/index.md:18]:
```suggestion
In addition to those, there are a lot of other community-supported agents:
```

File: docs/developer-docs/build/cdks/index.md [hunk 3]
```diff
@@ -1,10 +1,28 @@
-# Develop using different languages
+# Canister Development Kits
 
-Because the Internet Computer supports dapps compiled to standard WebAssembly modules, you can use many different programming languages to create your applications. The two officially supported languages of the Internet Computer are [Motoko](./motoko) and [Rust](./rust/rust-intro.md). Motoko is specifically designed to support the unique features of the Internet Computer and to provide a familiar yet robust programming environment. Rust is a powerful and type sound modern programming language with an active developer community. Of course, developers are not required to work in one (or both) of those languages. There are many different [community-supported languages](./other-languages/other-languages-intro.md) available as well.
+To create canisters, it is typical to use a so-called CDK, a Canister Development Kit.
+The CDK's main tasks are to expose the public interface of a canister, manage memory, allowing other canisters to be called, and to interface with the system API.
```
Comment (ericswanson-dfinity) [inline docs/developer-docs/build/cdks/index.md:4]: verb tense agreement
```suggestion
The CDK's main tasks are to expose the public interface of a canister, manage memory, allow other canisters to be called, and to interface with the system API.
```

**Review by ericswanson-dfinity — APPROVED** [review]

**Pull request merged**